conf/server.properties

```diff
@@ -110,14 +110,3 @@ jcs.default.cacheattributes.MaxObjects = 5000
 # Configuration for SparqlFilter
 org.dbpedia.spotlight.sparql.endpoint = http://dbpedia.org/sparql
 org.dbpedia.spotlight.sparql.graph = http://dbpedia.org
-
-# Topical classification properties
```

Comment:
I think that instead of removing these lines, it would be good to comment them. I remember some old GSoc project using that.  Maybe @dirkweissenborn has something to say on this ?